src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/Manifest.cs

```diff
@@ -51,6 +51,11 @@ public Manifest(ManifestMetadata metadata, ICollection<ManifestFile> files)
         public bool HasFilesNode { get; }
 
         /// <summary>
+        /// Gets the resolved manifest XDocument populated after save
+        /// </summary>
+        public XDocument ResolvedManifest { get; private set; }
```

Comment:
same here, why is this needed as a property?

Reply:
Removing, not needed anymore